### PR TITLE
N25: Sombervale

### PR DESCRIPTION
**Pull request by parasyte**
_Part of #751_

### CODE REVIEW

File: content/news/025/index.md [hunk 1]
```diff
@@ -67,6 +67,36 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [Sombervale]
+
+![Screenshot of Sombervale depicting the starting location](./sombervale.png)
+
+Sombervale by [@blipjoy] is a game built in 7 days for Rusty Jam. It is styled
```
Comment (17cupsofcoffee) [inline content/news/025/index.md:74]: The title and username links don't have a corresponding URL at the bottom of the section.

Comment (parasyte) [reply content/news/025/index.md:74]: Thank you! Should be resolved now.

**Review by AngelOnFira — APPROVED** [review]

**Comment by AngelOnFira** [comment]
Please rebase or allow maintainers to edit your fork :slightly_smiling_face:

**Comment by parasyte** [comment]
@AngelOnFira I have rebased. I'm not able to enable the "allow edits" feature because there is a bug in GitHub: https://github.com/isaacs/github/issues/1681 But I'll keep on top of this until you can merge it, or just move the fork to my personal account so I can enable that feature...

**Pull request merged**